src/schemas/intents.json

```diff
@@ -609,6 +609,33 @@
                             "properties": {
                                 "query": {
                                     "type": "string"
+                                },
+                                "suggestions": {
+                                    "type": "array",
+                                    "properties": {
+                                        "entity": {
+                                            "type": "object",
+                                            "items": {
+                                                "anyOf": [
+                                                    {
+                                                        "$ref": "https://meta.comcast.com/firebolt/entity#/definitions/MovieEntity"
```

Comment:
You can use `ProgramEntity` which is an already defined list of these entities.
But how come searching can only suggest programs? Seems like Channels and Music makes sense to suggest too?